src/lesson10/exceprions/userValidationService/UserValidator.java

```diff
@@ -0,0 +1,38 @@
+package lesson10.exceprions.userValidationService;
+
+public class UserValidator {
+
+    private void assertUserNotNull(User user) {
+        if (user.getFirstName() == null || user.getLastName() == null) {
```

Comment:
`User` object itself may be null

Reply:
does it mean you need to override the default constructor?

Reply:
No, you should add the check here `user == null`

Reply:
added